src/TheThingsNetwork.cpp

```diff
@@ -7,9 +7,12 @@
 #define debugPrintLn(...) { if (debugStream) debugStream->println(__VA_ARGS__); }
 #define debugPrint(...) { if (debugStream) debugStream->print(__VA_ARGS__); }
 
-void TheThingsNetwork::init(Stream& modemStream, Stream& debugStream) {
-  this->modemStream = &modemStream;
-  this->debugStream = &debugStream;
+void TheThingsNetwork::init(Stream& modemStream) {
+  this->modemStream = &modemStream; 
```

Comment:
Since we now have setModemStream as well we can remove this argument as well and in init error if modemstream wasn't set already